src/lp_matrix_data.jl

```diff
@@ -0,0 +1,192 @@
+#  Copyright 2017, Iain Dunning, Joey Huchette, Miles Lubin, and contributors
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+    LPMatrixData{T}
+
+The struct returned by [`lp_matrix_data`](@ref). See [`lp_matrix_data`](@ref)
+for a description of the public fields.
+"""
+struct LPMatrixData{T}
+    A::SparseArrays.SparseMatrixCSC{T,Int}
+    b_lower::Vector{T}
+    b_upper::Vector{T}
+    x_lower::Vector{T}
+    x_upper::Vector{T}
+    c::Vector{T}
+    c_offset::T
+    sense::MOI.OptimizationSense
+    variable_to_column::Dict{GenericVariableRef{T},Int}
```

Comment:
Isn't it a bit inconsistent to have a column -> `VariableRef` map but a row -> `ConstraintRef` map ?

Reply:
That's fair. Although at that point we could just direct people to `all_variables(model)`.

Reply:
Updated